backup.go

```diff
@@ -259,6 +259,6 @@ func (db *DB) Load(r io.Reader, maxPendingWrites int) error {
 	if err := ldr.Finish(); err != nil {
 		return err
 	}
-	db.orc.txnMark.Done(db.orc.nextTxnTs - 1)
+	db.orc.txnMark.SetDoneUntil(db.orc.nextTxnTs - 1)
```

Comment:
This should be `txnMark.Done(..)`. What if there other transactions running at the `ts<db.orc.nextTxnTs`? We would mark transactions as done which haven't completed yet.

Reply:
All the cases where I have changed `Done()` into `SetDoneUntil()` are cases where `Done()` was called without a corresponding `Begin()`, as far as I know.

Reply:
Is it possible to not make this change and still use your implementation? I understand that the existing behavior wasn't perfect but if possible I'd like to avoid this change since it is hard to test this.